### ISSUE TITLE
Install on 28-01-2021 of genie w/ pyats has all logging message sent to STDOUT

### ISSUE THREAD

**Issue by masadaco** [CiscoTestAutomation/pyats#75]
I have a semi-large application utilizing Netmiko to collect Cisco device information currently with TextFSM templates.  I decided to switch to `genie` and `pyats` to take advantage of Netmiko's `ConnectHandler(**kwargs)` with `.send_command("show version", use_genie=True)`.  The packages work beautifully.  However...

Now all logging messages are sent to my terminal (macOS).  I cannot find where a log handler is even being defined or any configuration settings in pyats documentation to control default console logging behavior.  There was no STDOUT logging prior to using pip install genie, pyats.  I have tried to configure logging to make StreamHandler() and ScreenHander() only 'critical' level, but nothing stops the INFO level console logs.

For example when I initialize an ELK client object...

```
In [2]: bob = refresh_elk()

In [3]: bob.info()
INFO:elasticsearch:GET http://elk.s.cloud:9200/ [status:200 request:0.188s]
```

Python 3.8.6 (default, Nov 20 2020, 18:29:40) 
pyats==21.1
pyats.aereport==21.1
pyats.aetest==21.1
pyats.async==21.1
pyats.connections==21.1
pyats.datastructures==21.1
pyats.easypy==21.1
pyats.kleenex==21.1
pyats.log==21.1
pyats.reporter==21.1
pyats.results==21.1
pyats.tcl==21.1
pyats.topology==21.1
pyats.utils==21.1
netmiko==3.3.2
genie==21.1
genie.libs.clean==21.1
genie.libs.conf==21.1
genie.libs.filetransferutils==21.1
genie.libs.health==21.1
genie.libs.ops==21.1
genie.libs.parser==21.1
genie.libs.sdk==21.1

Any advice or something I can set to turn off STDOUT logging.  Thanks!

**Comment by masadaco** [comment]
Additional info... I tried installing genie / pyats on a RHEL7 host with python3.6 and got the same impact.  All logging from some unknown logger to STDOUT.

**Comment by masadaco** [comment]
More info... console logging only appears after loading `netmiko` with `pyats` and `genie`

```
In [8]: import pyats
In [9]: elk_network.info()
Out[9]: 
{'data': 'hidden'}
In [10]: import genie
In [11]: elk_network.info()
Out[11]: 
{'data': 'hidden'}
In [12]: import netmiko
In [13]: elk_network.info()
INFO:elasticsearch:GET http://elk.s.cloud:9200/ [status:200 request:0.095s]
Out[13]: {'data': 'hidden'}
```

**Comment by masadaco** [comment]
A lot of troubleshooting later...

```python
import netmiko
netmiko.logging.disable()
```

This eliminated the STDOUT logging for me, but I'm still not sure how installing `pyats` and `genie` triggered this logging change.

**Comment by ktbyers** [comment]
FYI, I ran into this issue also when I accidentally installed pyats/genie `21.1` (instead of pyats `21.10`...dang YAML and strings/floats).

The problem went away when I upgraded to pyats/genie `21.10`.

I am pretty sure that pyats/genie was doing something with logging that was causing the stdout logging change and the Netmiko part was a side-effect (i.e. Netmiko was importing pyats and genie if they are installed, and those imports of pyats/genie were changing the logging behavior).
